cli/hoodie-defaults.js

```diff
@@ -9,6 +9,7 @@ function getHoodieDefaults () {
     dbUrl: undefined,
     inMemory: false,
     loglevel: 'warn',
-    url: undefined
+    url: undefined,
+    applicationRoot: undefined
```

Comment:
yes to both, we can’t decouble Hoodie from Hapi right now, and we don’t need to worry about a global `hoodie-cli` installation at this point.